docrawl/docrawl_client.py

```diff
@@ -116,16 +118,21 @@ def _wait_until_function_is_done(self, timeout):
             # docrawl_logger.info('Function is still running, waiting 0.5 sec ...')
 
         if is_function_done:
-            docrawl_logger.success('Function finished')
+            if function_error_message is None:
+                docrawl_logger.success('Spider function finished successfully')
+            else:
+                docrawl_logger.error(f'Spider function failed: {function_error_message}')
+                raise SpiderFunctionError(spider_function['error'])
         else:
             docrawl_logger.error('Function was not finished')
+            raise TimeoutError('Spider function timed out')
 
     def _execute_function(self, function, function_input=None, timeout=30):
-        docrawl_logger.info(f'Running function {function} with input: {function_input}')
+        # docrawl_logger.info(f'Running function {function} with input: {function_input}')
```

Comment:
If you're tired of logs, change the verbosity - I needed them for debugging - will add them back anyway